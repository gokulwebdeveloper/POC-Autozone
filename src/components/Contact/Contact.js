import React, { Fragment } from "react";
import GoogleMap from "./Map/GoogleMap";
import Data from "./MockData/Mock.json";
import { Helmet } from "react-helmet";

export default function Contact(props) {
  return (
    <Fragment>
      <Helmet>
        <meta charset="utf-8"/>
        <title>Autozone Contact</title>
        <meta name="description" content="This is Autozone contact page information"/>
      </Helmet>
      <div className="container-fluid">
      <div className="row">
        <div className="col-sm-7 col-md-7 px-0 common-col common-col-1">
          <div className="min-vh-100 common-col-1-contents">
            <GoogleMap />
          </div>
        </div>
        <div className="align-items-center justify-content-center d-flex col-sm-5 col-md-5 px-0 common-col common-col-2">
          <div className="justify-content-center m-5 p-0 common-col-1-contents">
            <h2>{Data.headerText}</h2>
            <p>{Data.para1}</p>
            <div className="address">
              <p>
                <span className="font-weight-bold">{Data.spanBoldText1}</span>
                {Data.para2}
              </p>
              <p>
                <span className="font-weight-bold">{Data.spanBoldText2}</span>
                {Data.para3}
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    </Fragment>
  );
}
