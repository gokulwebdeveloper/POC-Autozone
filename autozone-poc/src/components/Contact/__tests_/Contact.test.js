import React from "react";
import { unmountComponentAtNode } from "react-dom";
import "@babel/polyfill";
import { shallow } from "enzyme";
import Contact from "../Contact";

let container = null;

beforeEach(() => {
  // Setting up Dom element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // Cleanup on exit
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

const wrapper = shallow(<Contact />);
describe("tests for Contact.js", () => {
  it("renders the comoponent", () => {
    expect(wrapper).toBeTruthy();
  });

//   it("should render the button", () => {
//     const button = wrapper.find("button");
//     const buttonText = button.text();
//     expect(buttonText).toBe("Logout");
//   });

//   it("should render the h1", () => {
//     const label = wrapper.find("h1");
//     const labelText = label.text();
//     expect(labelText).toBe("Image Gallery");
//   });
});
