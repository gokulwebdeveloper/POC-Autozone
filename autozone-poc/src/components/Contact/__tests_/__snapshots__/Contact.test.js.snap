// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for Contact.js SnapShot Match 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-sm-7 col-md-7 px-0 common-col common-col-1"
    >
      <div
        className="min-vh-100 common-col-1-contents"
      >
        <div>
          Loading...
        </div>
      </div>
    </div>
    <div
      className="align-items-center justify-content-center d-flex col-sm-5 col-md-5 px-0 common-col common-col-2"
    >
      <div
        className="justify-content-center m-5 p-0 common-col-1-contents"
      >
        <h2>
          How to Find Us
        </h2>
        <p>
          Mauris viverra cursus ante laoreet eleifend. Donec vel fringilla ante. Aenean finibus velit id urna vehicula, nec maximus est sollicitudin.
        </p>
        <div
          className="address"
        >
          <p>
            <span
              className="font-weight-bold"
            >
              Address:
            </span>
             
            10 Suffolk st Soha, New Delhi, India
          </p>
          <p>
            <span
              className="font-weight-bold"
            >
              Telephone:
            </span>
             
            +12 34 567 890
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
